index.html

```diff
@@ -4053,6 +4074,34 @@ <h3>Add HTML Defaults</h3>
 					</ol>
 				</section>
 			</section>
+
+			<section id="processing-pub-resources">
+				<h3>Obtain a List of Publication Resources</h3>
+
+				<p>To <dfn>obtain a list of publication resources</dfn> from the <a>internal representation</a> of the
+					manifest <var>internal_rep</var> generated from running <a>processing the manifest</a>, run the
+					following steps:</p>
+
+				<ol>
+					<li>
+						<p>Let <var>resources</var> be an empty <a href="https://infra.spec.whatwg.org/#ordered-set"
+								>ordered set</a>.</p>
+					</li>
+					<li>
+						<p><a href="https://infra.spec.whatwg.org/#set-iterate">For each</a>
+							<var>resource</var> of <var>internal_rep["readingOrder"]</var>, <a>append</a>
+							<var>resource["url"]</var> to <var>resources</var>.</p>
+					</li>
+					<li>
+						<p><a href="https://infra.spec.whatwg.org/#set-iterate">For each</a>
+							<var>resource</var> of <var>internal_rep["resources"]</var>, <a>append</a>
+							<var>resource["url"]</var> to <var>resources</var>.</p>
+					</li>
+					<li>
+						<p>Return <var>resources</var>.</p>
```

Comment:
This would make the original `resources` list unknowable (per this description) from the internal representation. The internal representation would only ever have the union.

Reply:
Right now it's not modifying the internal representation; it's simply a means of extracting the list from the two properties (the variable `resources` is not the same as the resource list).

If we want to add the result into the internal representation, then, yes, we'd need a different term to store it under to avoid clobbering the existing property that uses that name. I'm not sure if it's necessary to store the list, though, or if it's only something the user agent might want to generate when needed. I don't have a strong opinion on this list, so would be interested to hear what others think.

Reply:
If it helps, I switched the variable name to `publication_resources` to avoid confusion with the existing manifest term.

Reply:
I would actually be in favor of adding this entry to the processing part as an extra term in the internal format, and refer to this algorithm at the very end of processing.